contracts/wrapperContracts/WrapperBase.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity 0.4.18;
+
+
+
+import "../Withdrawable.sol";
+import "../PermissionGroups.sol";
+
+
+contract WrapperBase is Withdrawable {
+
+    PermissionGroups wrappedContract;
+
+    struct DataTracker {
+        address [] approveSignatureArray;
+        uint lastSetNonce;
+    }
+
+    DataTracker[] internal dataInstances;
+
+    function WrapperBase(PermissionGroups _wrappedContract, address _admin) public {
```

Comment:
maybe number of data instances should be ctor param.

Reply:
yes. good idea